kolibri/core/assets/src/vue/core-base.vue

```diff
@@ -35,7 +35,7 @@
   const values = require('lodash.values');
   const responsiveWindow = require('kolibri.coreVue.mixins.responsiveWindow');
 
-  const PADDING = 32;
+  const PADDING = 16;
```

Comment:
note for later - just want to cycle back and make sure this change is correct. I think it might have affected some instances that were supposed to be 32px